services/ui-src/src/components/modals/AddEditReportModal.tsx

```diff
@@ -90,6 +93,7 @@ export const AddEditReportModal = ({
       formData["reportingPeriodEndDate"]
     );
     const programIsPCCM = formData["programIsPCCM"];
+    const ilosAvailable = ilos;
```

Comment:
Is this `const` needed? Could assign to `ilos` to `ilosAvailable` directly in the metadata hash.

Reply:
good point! removed it.